src/block-cache.ts

```diff
@@ -156,80 +157,82 @@ export function createBlockCacheMiddleware({
     fork: blockCache,
   };
 
-  return createAsyncMiddleware(async (req, res, next) => {
-    // allow cach to be skipped if so specified
-    if ((req as JsonRpcRequestToCache).skipCache) {
-      return next();
-    }
-    // check type and matching strategy
-    const type: string = cacheTypeForMethod(req.method);
-    const strategy: BlockCacheStrategy = strategies[type];
-    // If there's no strategy in place, pass it down the chain.
-    if (!strategy) {
-      return next();
-    }
+  return createAsyncMiddleware(
+    async (req: JsonRpcRequestToCache<string[]>, res, next) => {
```

Comment:
I'd recommend using "hide whitespace" when viewing this diff; most of these changes are just indentation resulting from the change on this line